lib/fs/ntfs/attrib_index_root.rb

```diff
@@ -91,15 +91,12 @@ def allocations=(indexAllocations)
       if @indexNodeHeader.hasChildren? && indexAllocations
         indexAllocations.each { |alloc| @indexAllocRuns << [alloc.header, alloc.data_run] }
       end
-      @indexAllocRuns
     end
 
     def bitmap=(bmp)
       if @indexNodeHeader.hasChildren?
         @bitmap = bmp.data.unpack("b#{bmp.length * 8}") unless bmp.nil?
       end
-
-      @bitmap
```

Comment:
We never check the return value of this method?

Reply:
And finally here - I think since the returned value is getting set to @bitmap it seemed to be overkill since its also getting set internally.  But that doesn't seem right either.  Time to revisit this as well.  Let me also find the CodeClimate complaint that had me address this and the previous one.

Reply:
So the complaint by CodeClimate on the @bitmap and @indexAllocRuns issues is "Variable `@indexAllocRuns` used in void context."  I probably didn't totally understand the message - but it seems that since the variable is being set in the method, and then returned to set the same variable, there is some sort of overkill happening.  I apparently don't really understand this and should undo this two commits.